lib/amazon-pricing/definitions/instance-type.rb

```diff
@@ -197,7 +199,7 @@ def self.get_values(json, category_type, override_price = false)
       'hi1.4xlarge' => 'High I/O Quadruple Extra Large',
       'cg1.4xlarge' => 'Cluster GPU Quadruple Extra Large',
       'cc1.4xlarge' => 'Cluster Compute Quadruple Extra Large', 'cc2.8xlarge' => 'Cluster Compute Eight Extra Large',
-      't1.micro' => 'Micro',
+      't1.micro' => 'T1 Micro Instance',
```

Comment:
curious: what is motivating this name change?

Reply:
trying to bring more in line with convention; looks like a change that I had made previously and got added inadvertently. I would argue that it's good though, as it provides more detail than 'micro' does.